test/e2e-test-application/e2e/tests/0-fiddle/fiddle-navigation.spec.js

```diff
@@ -337,4 +337,88 @@ describe('Fiddle', () => {
       });
     });
   });
+
+  describe('Theming', () => {
+    let newConfig;
+    beforeEach(() => {
+      newConfig = cloneDeep(fiddleConfig);
+      newConfig.settings.theming = {
+        themes: () => [
+          { id: 'light', name: 'Fiori3 Light' },
+          { id: 'dark', name: 'Fiori3 Dark' }
+        ],
+        defaultTheme: 'light'
+        // nodeViewURLDecorator: {
+        //   queryStringParameter: {
+        //     keyName: 'sap-theme'
+        //     // optional
+        //     // value: themeId => {
+        //     //   return themeId + 'YES';
+        //     // }
+        //   }
+        // }
+      };
+    });
+    it('Client get and set theme', () => {
+      cy.visitWithFiddleConfig('/', newConfig);
+
+      cy.window().then(win => {
+        cy.getIframeWindow().then(win => {
+          const defaultTheme = win.LuigiClient.uxManager().getCurrentTheme();
+          expect(defaultTheme).to.equal('light');
+
+          // not yet implemented
+          // win.LuigiClient.uxManager().setCurrentTheme('dark');
+          // expect(defaultTheme).to.equal('dark');
+        });
+      });
+    });
+    it('Client get and set theme', () => {
+      cy.visitWithFiddleConfig('/', newConfig);
+
+      cy.window().then(win => {
+        cy.getIframeWindow().then(win => {
+          const defaultTheme = win.LuigiClient.uxManager().getCurrentTheme();
+          expect(defaultTheme).to.equal('light');
+
+          // not yet implemented
+          // win.LuigiClient.uxManager().setCurrentTheme('dark');
+          // expect(defaultTheme).to.equal('dark');
+        });
+      });
+    });
```

Comment:
looks like there are 2 similar tests, maybe the one was planned to go through false-negative results?